docs/versions

```diff
@@ -0,0 +1 @@
+gamarin/docs-update
```

Comment:
does this need to change later to `master` or a release tag?

Reply:
mircea still needs to push a final change